assets/src/features/shared/components/BlockingScreen.tsx

```diff
@@ -0,0 +1,20 @@
+import clsx from "clsx";
+import React from "react";
+import RotateRight from "../../room-page/icons/RotateRight";
+
+const BlockingScreen: React.FC<{ message?: string }> = ({ message }) => {
+  return (
+    <div
+      className={clsx(
+        "absolute inset-0 z-30 flex h-full w-full flex-col items-center justify-center gap-y-4 bg-brand-sea-blue-100 font-aktivGrotesk text-brand-dark-blue-500",
+        "bg-videoroom-background bg-contain bg-center bg-no-repeat"
```

Comment:
Why do `room-page` need a custom CSS class with background but the blocking screen can have only tailwind classes? Can we unify this?